### ISSUE TITLE
dbt.profilesDirOverride does not resolve relative paths

### ISSUE THREAD

**Issue by brandon-segal** [innoverio/vscode-dbt-power-user#180]
### Expected behavior

I would expect that I could set the profiles directory for a dbt projects code-workspace file using something like 
```json
        "dbt.profilesDirOverride":".",
        "terminal.integrated.env.osx": {
            "DBT_PROFILES_DIR": "${workspaceFolder}"
        },
```
Or 
```json
        "dbt.profilesDirOverride":"${workspaceFolder}",
        "terminal.integrated.env.osx": {
            "DBT_PROFILES_DIR": "${workspaceFolder}"
        },
```
to have it in both my integrated terminal and my dbt extension tasks. 

### Actual behavior

When I run a compile operation or something similar with a relative path or one defined with variables I get the following error:

> Executing task: dbt compile --model dim_groups
[0m23:49:09  Encountered an error while reading the project:
[0m23:49:09    ERROR: Runtime Error
  Could not find profile named 'my-profile-I-define-in-my-profiles-yaml'
[0m23:49:09  Encountered an error:
Runtime Error
  Could not run dbt
Traceback (most recent call last):
...
    raise dbt.exceptions.RuntimeException("Could not run dbt") from exc
dbt.exceptions.RuntimeException: Runtime Error
  Could not run dbt

Currently this issue is resolved by passing a absolute path. This is however not sharable to others machines and would require additional work of creating a dev container for dbt development where the profile would be mounted to a known container location. 

### Steps To Reproduce

Create a dbt project with a profiles.yml defined in the same depth as the dbt_project.yml and create and activate a code workspace that defines the following settings: 
```json
        "dbt.profilesDirOverride":".",
        "terminal.integrated.env.osx": {
            "DBT_PROFILES_DIR": "${workspaceFolder}"
        },
```

### Operating System

OS 12.6

### dbt version

1.2.0

### dbt Adapter

dbt-bigquery

### dbt Power User version

0.7.9

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

**Comment by mdesmet** [comment]
This issue is a known issue that we currently don't parse all vscode variables.

Can you investigate how this is done in the Python vscode extension?

**Comment by mdesmet** [comment]
`"dbt.profilesDirOverride":".",` should work now. Just fixed an issue in 0.8.2

Still would be interesting to investigate how the python extension handles the evironment variables when running or debugging something.
